exonum/src/events/network.rs

```diff
@@ -113,8 +122,12 @@ impl ConnectionPool {
         }
     }
 
-    fn len(&self) -> usize {
-        self.peers.borrow().len()
+    fn count_outgoing(&self) -> usize {
+        let peers = self.peers.borrow();
+        peers
+            .iter()
+            .filter(|(_, e)| !e.address.is_incoming())
```

Comment:
I wonder why have you removed `is_outgoing` in order to use `!is_incoming()` here. It would be more direct to leave `is_outgoing` and remove `is_incoming` instead.